"use strict";(self.webpackChunkpadel_app=self.webpackChunkpadel_app||[]).push([[2190],{52190:e=>{e.exports=JSON.parse('{"default":{"success":"Processo conclu\xeddo com sucesso","error":"Algo deu errado.","empty":"Ainda n\xe3o h\xe1 dados para mostrar."},"login":{"401":"Falha no login. O nome de usu\xe1rio ou senha fornecidos n\xe3o correspondem aos nossos registros.","error":"Falha no login devido a um erro interno, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."},"activityLogTable":{"empty":"Ainda n\xe3o h\xe1 conte\xfado para mostrar","noMatch":"N\xe3o h\xe1 resultados para sua busca","error":{"title":"Falha ao carregar","description":"O log de atividades n\xe3o p\xf4de ser carregado devido a um erro interno. Por favor, tente novamente mais tarde."}},"customersTable":{"empty":"Ainda n\xe3o h\xe1 clientes para mostrar.","noMatch":"N\xe3o h\xe1 resultados para sua busca","error":{"title":"Falha ao carregar","description":"A lista de clientes n\xe3o p\xf4de ser carregada devido a um erro interno. Por favor, tente novamente mais tarde."}},"alerts":{"empty":"N\xe3o h\xe1 alertas","error":{"title":"Falha ao carregar","description":"A lista de alertas n\xe3o p\xf4de ser carregada devido a um erro interno. Por favor, tente novamente mais tarde."}},"createUser":{"successTitle":"Usu\xe1rio criado com sucesso","success":"Por favor, envie as credenciais para o novo usu\xe1rio.","errorTitle":"Erro interno","error":"O operador n\xe3o p\xf4de ser criado, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."},"editUser":{"successTitle":"O usu\xe1rio {{userId}} foi editado com sucesso","success":"Por favor, envie as novas credenciais para o usu\xe1rio.","errorTitle":"Erro interno","error":"O usu\xe1rio n\xe3o p\xf4de ser editado, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."},"deleteUser":{"success":"O usu\xe1rio {{userId}} foi exclu\xeddo com sucesso","errorTitle":"Erro interno","error":"O usu\xe1rio n\xe3o p\xf4de ser editado, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."},"dismissAll":{"error":"Os alertas n\xe3o puderam ser descartados, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."},"dismiss":{"error":"O alerta n\xe3o p\xf4de ser descartado, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."},"operatorNewCustomer":{"success":"O cliente {{customer}} foi criado com sucesso.","errorTitle":"Falha ao carregar","error":"O cliente {{customer}} n\xe3o p\xf4de ser criado, por favor tente novamente. Se o erro persistir, entre em contato com nossa equipe de suporte."}}')}}]);